PMS_Updater.sh

```diff
@@ -6,7 +6,13 @@ VERBOSE=1
 REMOVE=1
 LOGGING=1
 
-PLEXTOKEN="$(sed -n 's/.*PlexOnlineToken="//p' /Plex\ Media\ Server/Preferences.xml | sed 's/\".*//')"
+#check if we're in pms plugin, or standard jail
+if [ -e "/Plex Media Server/Preferences.xml" ]; then {
+        PREFS="/Plex Media Server/Preferences.xml"
+} else {
+        PREFS="/usr/local/plexdata-plexpass/Plex Media Server/Preferences.xml"
```

Comment:
In order to support non-plexpass (`-n`) can you swap `plexdata-plexpass` to `$PMSDATAFOLDER` which is set in the if block on 207

```sh
PMSDATAFOLDER="plexdata-plexpass"
...
PMSDATAFOLDER="plexdata"
```

Reply:
@mweber03 I don't see PMSDATAFOLDER anywhere, I do see $PMSPARENTPATH/$PMSLIVEFOLDER ?  I could perhaps leverage that.  I might not be looking at the same version of the file though.